R/generate_template.R

```diff
@@ -85,7 +90,7 @@
 #' create_new_application(name = 'mytestapp', location = tempdir())
 #'
 #' @export
-create_new_application <- function(name, location, sampleapp = FALSE) {
+create_new_application <- function(name, location, sampleapp = FALSE, right_sidebar = FALSE) {
```

Comment:
for consistency I would like to rename the parameter to rightsidebar - there is no underscore in the existing sampleapp parameter.  Just the user-facing parameter needs to be renamed, usage internally is fine.  Be sure to change any documentation instances (examples, etc) also